tests/functional/deployment/test_storagecluster_deletion.py

```
@@ -0,0 +1,91 @@
+        sc_name = constants.DEFAULT_CLUSTERNAME
+
+        storage_cluster = ocp.OCP(
+            kind=constants.STORAGECLUSTER,
+            resource_name=sc_name,
+            namespace=ns_name,
+        )
+
+        annotations = storage_cluster.get().get("metadata", {}).get("annotations", {})
+        original_value = annotations.get(constants.CONFIRM_DELETION_ANNOTATION)
+
+        def restore_annotation():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `restore_annotation`

Add return type annotation: `None`

(ANN202)